lib/options.js

```diff
@@ -350,6 +350,13 @@ module.exports = function(usingFlatConfig) {
                 description: "Chooses severity level for reporting unused eslint-disable and eslint-enable directives",
                 enum: ["off", "warn", "error", "0", "1", "2"]
             },
+            {
+                option: "report-unused-inline-configs",
+                type: "String",
+                default: void 0,
+                description: "Adds reported errors for unused eslint-disable and eslint-enable directives",
```

Comment:
This description doesn't seem to match the behavior of this option. Also, this option should be available only in flat config mode (see, for example, `warnIgnoredFlag` in this file).